src/main/java/com/example/manager/AsdUserManagerApplication.java

```diff
@@ -1,13 +1,181 @@
 package com.example.manager;
 
+import java.util.Date;
+import java.util.Scanner;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;
+
+import com.example.manager.domain.User;
+import com.example.manager.controller.UserController;
+
+@SpringBootApplication(exclude = {DataSourceAutoConfiguration.class})
+@EnableMongoRepositories
+public class AsdUserManagerApplication implements CommandLineRunner {
+
+    @Autowired
+    private UserController uc;
+    Scanner scn = new Scanner(System.in);
+    int cnt = 0;
```

Comment:
Naming could be improved - after looking at the code, I cannot determine, what "uc" does until scrolling up